app/services/update_plan_service.rb

```diff
@@ -236,6 +236,10 @@ def check_any_update
     return true
   end
 
+  # No longer validate the data release strategy on work plan dispatch,
+  # as the user dispatching the plan may not have the same permissions as the plan owner
+  # who originally selected to data release strategy
+
```

Comment:
Little confused by this comment as nothing else has changed in the file. Does the change need to happen? Or has it happened elsewhere?